pkg/formatted/task.go

```diff
@@ -0,0 +1,26 @@
+// Copyright © 2021 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package formatted
+
+import (
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+)
+
+func GetTaskRefName(task *v1beta1.PipelineTask) string {
+	if task.TaskRef != nil {
+		return task.TaskRef.Name
+	}
```

Comment:
can we add a test for this?

Reply:
it's explicit with https://github.com/tektoncd/cli/pull/1303/files#diff-37b860e7d4510beb113f11d5a6aa216ead6b58c455f495cb45c49ea71042d6bcR549 isnt it ?

i can add a implicit test but i don't think that add value or improve coverage,

Reply:
yeah I agree but I see that each entry in the formatted package has a corresponding test hence said in order to maintain the consistency

Reply:
Oh yeah fair enough, let maybe add that then..